searches/binary_search.py

```diff
@@ -298,8 +298,8 @@ def binary_search_by_recursion(
 
 
 if __name__ == "__main__":
-    user_input = input("Enter numbers separated by comma:\n").strip()
-    collection = sorted(int(item) for item in user_input.split(","))
+    user_input = input("Enter numbers separated by space\n")
```

Comment:
```suggestion
    user_input = input("Enter numbers separated by space\n").strip()
```
Why get rid of the `.strip()`? It'd help remove any trailing whitespace in the input.